components/ContributionConfirmationModal.js

```diff
@@ -0,0 +1,237 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useMutation } from '@apollo/client';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import { getCurrencySymbol } from '../lib/currency-utils';
+import { i18nGraphqlException } from '../lib/errors';
+import { API_V2_CONTEXT, gqlV2 } from '../lib/graphql/helpers';
+
+import Container from './Container';
+import FormattedMoneyAmount from './FormattedMoneyAmount';
+import { Box, Flex } from './Grid';
+import InputField from './InputField';
```

Comment:
`InputField` is deprecated and should be avoided for new developments. Please use the `Styled*` equivalents: `StyledInput`, `StyledInputAmount`, etc.

Reply:
It's a mistake that we haven't marked it as deprecated in the code sooner. Fixing that in https://github.com/opencollective/opencollective-frontend/pull/7275

Reply:
I've changed everything to remove `InputField`. 🙂